crates/re_space_view_dataframe/src/space_view_class.rs

```diff
@@ -189,3 +215,29 @@ impl SpaceViewClass for DataframeSpaceView {
         Ok(())
     }
 }
+
+/// Returns a sorted, deduplicated iterator of all instance paths for a given entity.
+///
+/// This includes _any_ instance key in all components logged under this entity path, excluding
+/// splats.
+fn sorted_instance_keys_for<'a>(
```

Comment:
```suggestion
fn sorted_instance_paths_for<'a>(
```

or change the return type